dual_quaternions_ros/dual_quaternions.py

```diff
@@ -238,9 +245,26 @@ def is_unit(self):
         """True if the norm of the rotation part of the dual quaternion is 1"""
         return np.allclose(self.q_r.norm(), 1.0)
 
-    def slerp(self, other, t):
-        """Spherical Linear Interpolation"""
-        raise NotImplementedError()
+    def pow(self, exponent):
+        """self^exponent"""
+        return self.exp(exponent * self.log(self))
+
+    @classmethod
+    def exp(cls, dq):
+        q_r = dq.q_r.exp()
+        q_d = q_r * dq.q_d
+        return cls(q_r, q_d)
+
+    @classmethod
+    def log(cls, dq):
+        q_r = dq.q_r.log()
+        q_d = dq.q_r.inverse() * dq.q_d
+        return cls(q_r, q_d)
+
+    @classmethod
+    def sclerp(cls, start, stop, t):
+        """Screw Linear Interpolation"""
+        return (stop * start.conjugate()).pow(t) * start
```

Comment:
```suggestion
        return start * (start.inverse() * stop).pow(t)
```
probably also want to check that t is in range [0, 1]